WordPress/Classes/ViewRelated/Blog/Blog Dashboard/Cards/Prompts/DashboardPromptsCardCell.swift

```diff
@@ -67,7 +67,7 @@ class DashboardPromptsCardCell: UICollectionViewCell, Reusable {
     private lazy var promptLabel: UILabel = {
         let label = UILabel()
         label.translatesAutoresizingMaskIntoConstraints = false
-        label.font = Style.promptContentFont
+        label.font = WPStyleGuide.BloggingPrompts.promptContentFont
```

Comment:
Just an FYI. The style names can get long, so sometimes we `typealias` them to something shorter, for example `private typealias Style = WPStyleGuide.BloggingPrompts`. You can do so if you like, either way is fine, just letting you know.